### DISCUSSION TITLE
友链 - Chris's blog

### DISCUSSION THREAD

**Discussion by giscus** [chrisyy2003/chrisyy2003.github.io#12] [评论]
# 友链 - Chris's blog

https://blog.chrisyy.top/links/

**Comment by t0hka1** [comment]
博客名：t0hka
简介：Reverse Engine/Blockchain noob
链接：https://blog.t0hka.top
图片：https://blog.t0hka.top/upload/2022/09/t0hka.jpg

**Reply by chrisyy2003** [reply]
好的师傅，我稍后添加

**Comment by WuhuVoV** [comment]
发现宝藏

**Comment by jaychois** [comment]
么么

**Reply by chrisyy2003** [reply]
😈😈😈
